src/test/java/com/cloudbees/plugins/credentials/CredentialsParameterDefinitionTest.java

```diff
@@ -157,6 +157,21 @@ public void escapeAndMarkupFormatAreDoneCorrectly() throws Exception {
         collector.checkThat("parameters page should not leave param name unescaped", text, not(containsString("<param name>")));
         collector.checkThat("parameters page should mark up param description", text, containsString("<b>[</b>param description<b>]</b>"));
         collector.checkThat("parameters page should not leave param description unescaped", text, not(containsString("<param description>")));
+
+        page = wc.getPage(p, "configure");
+        form = page.getFormByName("config");
+        HtmlTextArea value = form.getTextAreaByName("parameter.description");
```

Comment:
This fails now.

I think the test improvement is not really necessary anymore, as this PR now only changes the presentation of the field on the UI. So this PR seems ready to go once the changes to this file are reverted.

Reply:
Hi @daniel-beck,

Test improvement has been reverted.